### PR TITLE
feat: GraphQL application items

### PR DESCRIPTION
**Pull request by mayaisrani**

### CODE REVIEW

File: oneapp-server/dao/ApplicationItemsDao.js
Comment (ronaldheft-usds) [inline oneapp-server/dao/ApplicationItemsDao.js:33]: This just denotes in the database that a SNAP application is being made, correct?

Comment (mayaisrani) [reply oneapp-server/dao/ApplicationItemsDao.js]: correct. I generalized it below so that it can used in the future for all item types

Comment (ronaldheft-usds) [reply oneapp-server/dao/ApplicationItemsDao.js]: ++ on generalizing. We'll just want to make sure we document the value needed to indicate it's a SNAP application in the static docs for @mikepif, otherwise I'm sure it won't be obvious.

**Comment by ronaldheft-usds** [comment]
I think this is looking just about ready to merge with tests once #440 is merged.

**Comment by mayaisrani** [comment]
running into rebase issues, will add to Application.test.js in a following pull request

**Review by ronaldheft-usds — APPROVED** [review]
Looks great!

**Pull request merged**
